features/users/input_models.py

```diff
@@ -0,0 +1,72 @@
+"""Users feature input model"""
+from pydantic import BaseModel, EmailStr, Field, field_validator
+from email_validator import validate_email
+
+import re
+import configuration
+
+config = configuration.Config()
+
+
+class RegisterUserInputModel(BaseModel):
+    username: str = Field(min_length=3, max_length=30)
+    email: str = EmailStr
+    password: str = Field()
+
+    @field_validator('email', mode='after')
+    @classmethod
+    def validate_email(cls, email):
+        try:
+            validate_email(email)
```

Comment:
having the same name is misleading.
Change validator function name or import the whole email_validator package and call it here as email_validator.validate_email